lightning/src/ln/functional_tests.rs

```diff
@@ -857,378 +853,6 @@ fn test_update_fee() {
 	close_channel(&nodes[0], &nodes[1], &chan.2, chan.3, true);
 }
 
-#[test]
```

Comment:
Any chance the test moves could be done in a follow-up PR? I'm touch quite a few of them in a PR that I'm working on.

Reply:
Actually, of the moved tests, looks like I only touch `htlc_fail_async_shutdown`, so should be manageable.